templates/admin/citizen_project/members.html.twig

```diff
@@ -95,10 +88,12 @@
                                     <span class="label label-default">Membre</span>
                                 </td>
                                 <td>
-                                    <a href="{{ path('app_admin_citizenproject_change_privilege', {citizen_project: citizen_project.id, adherent: member.id, privilege: constant('CITIZEN_PROJECT_ADMINISTRATOR', membership), token: token}) }}"
-                                       class="btn btn-default btn-sm">
-                                        Transformer en administrateur
-                                    </a>
+                                    {% if citizen_project.status is same as(constant('APPROVED', citizen_project)) %}
```

Comment:
I think you can use this function here: ` {% if citizen_project.isApproved %}`

Reply:
:ok_hand: